www/core/j4/default.xml

```diff
@@ -5,21 +5,21 @@
 		<description>Joomla! 4.4 CMS</description>
 		<element>joomla</element>
 		<type>file</type>
-		<version>4.4.4</version>
-		<infourl title="Joomla 4.4.4 Release">https://www.joomla.org/announcements/release-news/5907-joomla-5-1-0-and-joomla-4-4-4-are-here.html</infourl>
+		<version>4.4.5</version>
+		<infourl title="Joomla 4.4.5 Release">https://www.joomla.org/announcements/release-news/5908-joomla-5-1-1-and-joomla-4-4-5-are-here.html</infourl>
 		<downloads>
-			<downloadurl type="full" format="zip">https://downloads.joomla.org/cms/joomla4/4-4-4/Joomla_4.4.4-Stable-Update_Package.zip</downloadurl>
-			<downloadsource type="full" format="zip">https://github.com/joomla/joomla-cms/releases/download/4.4.4/Joomla_4.4.4-Stable-Update_Package.zip</downloadsource>
-			<downloadsource type="full" format="zip">https://update.joomla.org/releases/4.4.4/Joomla_4.4.4-Stable-Update_Package.zip</downloadsource>
+			<downloadurl type="full" format="zip">https://downloads.joomla.org/cms/joomla4.4.5-5/Joomla_4.4.5-Stable-Update_Package.zip</downloadurl>
```

Comment:
that looks wrong the path should be https://downloads.joomla.org/cms/joomla4/4-4-5/Joomla_4.4.5-Stable-Update_Package.zip